docs/intents.rst

```diff
@@ -202,6 +202,6 @@ On Windows use ``py -3`` instead of ``python3``.
 
 .. warning::
 
-    There is no currently set date in which the old gateway will stop working so it is recommended to update your code instead.
+    There is no current set date on which the old gateway will stop working so it is recommended to update your code instead.
```

Comment:
```suggestion
There is no current set date on when the old gateway will stop working so it is recommended to update your code instead.
```

Reply:
thats not grammatically correct, either `date on which ...` or `date for when ...`

Reply:
Agreed `date on which` sounds the best

Reply:
Fair enough.